src/Castle.Core/DynamicProxy/Contributors/ClassProxyWithTargetTargetContributor.cs

```diff
@@ -58,7 +55,7 @@ protected override IEnumerable<MembersCollector> CollectElementsToProxyInternal(
 		protected override MethodGenerator GetMethodGenerator(MetaMethod method, ClassEmitter @class,
 		                                                      OverrideMethodDelegate overrideMethod)
 		{
-			if (methodsToSkip.Contains(method.Method))
+			if (method.Ignore)
 			{
 				return null;
 			}
```

Comment:
Such checks are missing in all interface target contributors, likely because they were never needed in the past. So this is a minor inconsistency, and perhaps we'll fix that someday... I chose to leave those other contributors as is for now.